src/commands/actions.js

```
@@ -156,6 +159,9 @@ module.exports.DeployActionCommand = class {
             }
 
             if (options.jobTimeout) {
+                if (actionInst.type === 'daemon') {
+                    printError('--jobTimeout not valid on daemon action types');
```

Thinking if we should throw an error or a warning. What would be better from a UX perspective?